mta-st.yaml

```diff
@@ -0,0 +1,95 @@
+_schema-version: '2.1'
+ID: bookshop-java-public
+version: 1.0.0
+description: "Bookshop CAP Java Project with UI"
+parameters:
+  enable-parallel-deployments: true
+modules:
+# --------------------- SERVER MODULE ------------------------
+  - name: bookshop-java-public-srv
+# ------------------------------------------------------------
+    type: java
+    path: srv
+    parameters:
+      memory: 1024M
+      disk-quota: 256M
+      buildpack: sap_java_buildpack
+    properties:
+      SPRING_PROFILES_ACTIVE: cloud
+    build-parameters:
+      builder: custom
+      commands:
+        - mvn clean package -DskipTests=true
+      build-result: target/*-exec.jar
+    requires:
+      - name: bookshop-java-public-hdi-container
+      - name: bookshop-java-public-uaa
+      - name: hdi_credentials
+        properties:
+          CDS_MULTITENANCY_DEPLOYER_USER: ~{user}
```

Comment:
Why do we have a CDS_MULTITENANCY configuration in the single tenant case?

Reply:
I removed it.